### ISSUE TITLE
Bug with pause/resume and onselectstart (textSelectionInterceptor)

### ISSUE THREAD

**Issue by douweknook** [anvaka/panzoom#68]
Hey,

First of all thanks for the great panzoom lib!

However, I think I found a bug with the pause/resume logic. The problem I ran into was that after pausing and resuming my panzoom instance I was no longer able to select any text in my app. This proved to be a problem for us as we use a rich text editor which relied on selection events to work. After some time debugging I found that after I call `panzoom.pause()`, it sets an event handler on `window.document.onselectstart` which looks like:
```
function disabled(e) {
  e.stopPropagation();
  return false;
}
```

However, after calling `panzoom.resume()` this disabled method is not removed, meaning text selection events on the entire window no longer work as expected. The default for `window.document.onselectstart` seems to be `null`. If I set the value back to `null` after I call `panzoom.resume()` everything seems to work as expected again but it feels like a bit of a hacky workaround.

As far as I can track it down in the package this logic happens added in the capture/release methods of the textSelectionInterceptor: https://github.com/anvaka/panzoom/blob/master/lib/textSelectionInterceptor.js
When this line: `window.document.onselectstart = prevSelectStart` gets executed it seems for some reason to store the `disabled` method in `prevSelectStart`.

I think the issue is caused because in our app we use a dragable element inside the panzoom element and it might affect the mouse up/down handlers to which the capture and release methods from textSelectionInterceptor are attached. I'm not 100% sure how to fix this bug (or if it should even be fixed in panzoom or in react-dnd which we use for creating dragable elements) but figured I'd make an issue to at least help people out who might run into this in the future :)

**Comment by nickjohnson** [comment]
I have a similar situation. I am simply trying to pause, do something, then resume. The resume doesn't bring functionality back.

**Comment by matsp** [comment]
There is an undocumented option "enableTextSelection" which fixes this behavior when set to true!

**Comment by anvaka** [comment]
published the fix in `v8.7.1`. Hopefully this resolves the original issue. Please reopen if it is still there.
